README.md

```diff
@@ -34,6 +34,22 @@ Last but not least, it also supports a [Volto Nodejs-based backend reference](ht
 demos how other systems could also use Volto to display and create content
 through it.
 
+## Demo
```

Comment:
@avoinea what do you think? We need to rewrite all the intro soon anyways and add the new Plone 6 demo site anyways.